ui/templates/base.html

```diff
@@ -58,6 +58,19 @@
         smartlook('init', '{{ APIKEYS.SMARTLOOK }}');
     </script>
     {% endif %}
+    <script type="text/javascript">
```

Comment:
I believe this template is applied to all pages, but the original ticket asks for it to only show up on 3. Is that going to be an issue?